src/data-types/bit.ts

```diff
@@ -14,7 +15,7 @@ module.exports = {
     buffer.writeUInt8(1);
   },
 
-  writeParameterData: function(buffer, parameter, options, cb) {
+  writeParameterData: function(buffer, parameter: ParameterData<null | unknown>, options, cb) {
```

Comment:
Let's stick to any here, until the remaining code was streamlined and it's clearer what the parameter data type can be.

```suggestion
  writeParameterData: function(buffer, parameter: ParameterData<any>, options, cb) {
```